pmd-core/src/main/java/net/sourceforge/pmd/cpd/TokenEntry.java

```diff
@@ -93,6 +114,16 @@ public int getBeginLine() {
         return beginLine;
     }
 
+    @Deprecated
+    public int getBeginColumn() {
```

Comment:
You have marked the new methods here already as deprecated. Can you describe please, how you envision the API to evolve here?
From the TODO comment, I understand, that this method should look like `public Optional<Integer> getBeginColumn()` in the future (possibly with PMD 7.0.0). But changing that signature would immediately break compilation, and right now there is no alternative to the deprecated method...